src/lamplib/src/genny/tasks/genny_runner.py

```diff
@@ -26,6 +28,22 @@ def main_genny_runner(
             raise Exception(f"genny_core not found at {path}.")
         cmd = [path, *genny_args]
 
-        run_command(
-            cmd=cmd, capture=False, check=True, cwd=workspace_root,
-        )
+        # Intercept the workload given to the core binary.
+        with tempfile.TemporaryDirectory() as tempdir:
```

Comment:
It would be nice if it was also possible to generate and store the post-processed files. Plus it would make life easier if the post-processed files were in the dsi diagnostics archive.

Otherwise it will be difficult to debug / develop new workloads and actors.

Reply:
I added the `./run-genny evaluate` command which has the same behavior as the `evaluate` subcommand currently part of the core binary, and could be piped to a file. It's also deterministic so there's no additional data from evergreen runs that would be needed if people can generate the file locally. Would that work for your use case?

Reply:
Ah nice. I was confused by the function name **create_new_actor**. That works for me (assuming it also gets added to the README) .

Reply:
Jim, for the purpose of debugging/developing new workloads, it is probably easier to use the new `./run-dsi evaluate` command that Raiden added. This way you don't need to run the actual workload to see what the processed yaml looks like. 

Nonetheless, it's not a bad idea to add the post-processed files to the artifacts, because the `evaluate` command only does minimal validation. If there is an error in the workload yaml, it would good to have the post-processed file to check exactly where that error is happening.

Reply:
Lurking. Fwiw any files in build/WorkloadOutput get included. Could just put it in there in a place that makes sense.

Reply:
Ooh cool that's the fancy conventional locations work results. Will try that, thanks!